### ISSUE TITLE
endosome to lysosome transport

### ISSUE THREAD

**Issue by bmeldal** [geneontology/go-ontology#12264]
Hi,

I'm trying to figure out which process terms are appropriate for VPS4 complexes which acts on the membrane fission of multivesicular bodies (MVBs), autophasosomes and lysosomes as well as plasma membrane fission at cytokinesis, viral budding and nuclear membrane fission.

1)
Why has the endosome not got a link to vacuole? The defs do not indicate a reason. 
We already have:
autolysosome is_a lysosome 
lysosome is_a vacuole 
amphisome is_a autophagosomes 
autophagosomes is_a vacuole 
All of these compartments can derive elements from the endosome which is also a membrane-enclosed, liquid-containing compartment.
Also:
multivesicular body is_a endosome
In fact, it's a late endosome, that link is also missing.
AI 1: 
replace multivesicular body is_a endosome
with
multivesicular body is_a late endosome

AI 2:
Add endosomal transport is_a vacuolar transport?

2)
I need a new process:
GO:NEW late endosome to lysosome transport via multivesicular body sorting pathway
is_a GO:0032510 endosome to lysosome transport via multivesicular body sorting pathway
is_a GO:0032511 late endosome to vacuole transport via multivesicular body sorting pathway
is_a GO:1902774 late endosome to lysosome transport

I don't know if there is a template as I need the "via" bit to distinguish it from GO:1902774 late endosome to lysosome transport

3)
I also think
GO:0032510 endosome to lysosome transport via multivesicular body sorting pathway
should be is_a GO:0008333 endosome to lysosome transport

Thanks,
Birgit

**Comment by paolaroncaglia** [comment]
Hi Birgit,

I'm assigning this to David H @ukemi . He's looked into related topics for the autophagy project and he'd probably have quicker answers. However, please note that both David and I will be on leave until mid-next week; let me know if this can't wait.
Also mentioning @marcfeuermann in case he wishes to comment.

@ukemi , feel free to assign back to me for edits.

Thanks,
Paola

**Comment by bmeldal** [comment]
Thank you all. Looking forward to hear back from you later next week.

**Comment by ukemi** [comment]
Hi Birgit. 
As a first step, I will make an endosome a vacuole and a MVB a late endosome. I will let the reasoner infer new relationships, which will include AI2. Once that is complete and I check to make sure that all looks ok, I will add your new term.
-D

**Comment by ukemi** [comment]
I also moved 
GO:0032510 endosome to lysosome transport via multivesicular body sorting pathway
to be is_a GO:0008333 endosome to lysosome transport instead of an GO:0045022 early endosome to late endosome transport.

**Comment by ukemi** [comment]
[Term]
+id: GO:0061764
+name: late endosome to lysosome transport via multivesicular body sorting pathway
+namespace: biological_process
+def: "The directed movement of substances from late endosomes to lysosomes by a pathway in which molecules are sorted into multivesicular bodies, which then fuse with the lysosome." [GOC:dph]
+is_a: GO:0032510 ! endosome to lysosome transport via multivesicular body sorting pathway
+is_a: GO:1902774 ! late endosome to lysosome transport
+created_by: dph
+creation_date: 2016-04-19T09:29:57Z

**Comment by bmeldal** [comment]
Thanks, David. I have updated the term. @Pauldenny the new term will appear on the website with the next release.

**Comment by ValWood** [comment]
H Birgiti,

I don't think an endosome is a lysosome?
An endosome is a transport vesicle https://en.wikipedia.org/wiki/Endosome
A lysosome is lytic.
See #12264

maybe confusing with GO:0036019 endolysosome? (which I only knew existed about 10 seconds ago!)

Val

**Comment by deustp01** [comment]
The distinction between vacuoles (and their is_a children) and vesicles (and theirs) looks pretty murky, and it looks like that's contributing to the problem here and in 12519:

GO:0031982 vesicle "Any small, fluid-filled, spherical organelle enclosed by membrane."

GO:0005773 vacuole "A closed structure, found only in eukaryotic cells, that is completely surrounded by unit membrane and contains liquid material. Cells contain one or several vacuoles, that may have different functions from each other. Vacuoles have a diverse array of functions. They can act as a storage organelle for nutrients or waste products, as a degradative compartment, as a cost-effective way of increasing cell size, and as a homeostatic regulator controlling both turgor pressure and pH of the cytosol."

From the definitions, it looks like a vacuole is_a vesicle, but looking at usage of child terms it's more nearly the case that vacuoles and vesicles are siblings, corresponding to different kinds of intracellular, membrane-bounded closed structures.

**Comment by bmeldal** [comment]
Val, 
I agree. Where does it say endosome is_a lysosome? Can't see it in this ticket.
Ontology has endosome is_a vacuole
Birgit

**Comment by ValWood** [comment]
In yeast the vacuole is equivalent to the lysosome, so I probably should have said 'vacuole' above.

Classifying an endosome as a vacuole is problematic for most species, but as Peter D' pointed out the  problems are a little more deep rooted.


This is  very vague...
A closed structure, found only in eukaryotic cells, that is completely surrounded by unit membrane and contains liquid material. Cells contain one or several vacuoles, that may have different functions from each other. Vacuoles have a diverse array of functions. They can act as a storage organelle for nutrients or waste products, as a degradative compartment, as a cost-effective way of increasing cell size, and as a homeostatic regulator controlling both turgor pressure and pH of the cytosol.

 Because it doesn't say these are all the vacuole functions, but if they are this doesn't fit endosomes...

v

**Comment by dosumis** [comment]
I suspect this has become confusing because of an attempt to reconcile various unrelated uses of the term 'vacuole'.  Perhaps we should ditch the general term vacuole and have only various specific types (plant-type, lytic ...).  

BTW - better to use reasoning to classify this term than manual classification:

+id: GO:0061764
+name: late endosome to lysosome transport via multivesicular body sorting pathway

This should still be possible even if we can't come up with a completely formalised definition.

**Comment by bmeldal** [comment]
Yes, I see the problem. The classic def for vacuole is along the lines of "some kind of membrane-enclosed organelle that contains liquid stuff' and there can be more than one in the cell and they can have lots of functions. Now, in yeast, they have a very precise function while in other eukaryotes they are a bucket term. Great :(

Solution: separate term for 'lytic' vacuoles in yeast?

I see DOS just said the same!

**Comment by ValWood** [comment]
I wondered about this and I think it makes sense if it's possible.

Even so, would it be correct to classify an endosome as a type of vacuole? An endosome is a transport vesicle, and a vacuole isn't?
Or is that different in too?

**Comment by bmeldal** [comment]
Based on my school-knowledge of vacuole I would have lumped everything into it and that's what the current def appears to refers to, too! 

I guess, if we redefine vacuole and vesicle properly, then endosome should become is_a vesicle :)

**Comment by dosumis** [comment]
I suspect not everyone comfortable with classifying endosome as transport
vesicle.  Perhaps best to keep directly under the class membrane bounded
organelle.  For me to be completely comfortable with vacuole as a general
class, I'd like to see a paper on its evolution, ideally with some
molecular evidence for a single origin.

**Comment by ukemi** [comment]
I agree with Birgit, my interpretation is that the term is used different ways by different communities and the animal biologists use it much more loosely. Yeast and plant biologists have a much tighter interpretation.

**Comment by bmeldal** [comment]
Looks like we are in agreement to redefine the defs for vacuole and vesicle

> I suspect not everyone comfortable with classifying endosome as transport vesicle. Perhaps best to keep directly under the class membrane bounded organelle. 
Ok, my knowledge of vesicles is too flaky to argue against a conservative approach for endosome!

> For me to be completely comfortable with vacuole as a general class, I'd like to see a paper on its evolution, ideally with some molecular evidence for a single origin.
Well, then we have to redefine it! At the moment it's a bucket term!

**Comment by ValWood** [comment]
Animal lysosomes and fungal vacuoles have common evolutionary origin (most of the  resident vacuolar complexes are conserved), which is  the evidence for a single origin...... No idea about plant vacuole origins....

@dosumis  so which type of endosome isn't a transport vesicle (endocytosis is_a vesicle-mediated transport?)

**Comment by ValWood** [comment]
I just did a quick scan. Pombe has 128 vacuolar membrane proteins, I only did membrane because the non-membrane proteins are likely to be largely  targets of vacuole degradation.

If these 105 have been identified in humna.
Conserved members include
V-type ATPase
HOPs/CORVET complex
Seh1 complex
autophagy pathway.
which are lysosomal in human?

(but at present it is also pulling in endosome membrane components like ESCRT I,II,III ...some of these *might* end up in vacuole membrane....)

**Comment by dosumis** [comment]
At least in metazoans, endocytic vesicle != endosome.  Endosome is a (set
of) relatively more stable compartments. Endocytic vesicles fuse with
(early) endosome. Sorting & targeting of components follows.

**Comment by dosumis** [comment]
So maybe general vacuole should be parent to lysosome, not endosome.  How
does the term lytic vacuole fit?

**Comment by ValWood** [comment]
I think I have been a bit sloppy with my annotations here ......
Need to look at annotations to endosome and /endocytic vesicle, but I 
suspect they will be largely overlapping.
Or maybe in yeast  there isn't much distinction between endosomes and 
endocytic vesicles because the cells are so much smaller.....I would 
need to check this out with SGD....(I'll follow up on this at some point 
soon-ish)

Still not heard of an endosome referred to as a vacuole, but I don't 
know how general that is.....



On 07/07/2016 13:33, David Osumi-Sutherland wrote:
> At least in metazoans, endocytic vesicle != endosome. Endosome is a (set
> of) relatively more stable compartments. Endocytic vesicles fuse with
> (early) endosome. Sorting & targeting of components follows.
> On 7 Jul 2016 12:54, "Val Wood" <notifications@github.com> wrote:
>
> > Animal lysosomes and fungal vacuoles have common evolutionary origin 
> (most
> > of the resident vacuolar complexes are conserved), which is the evidence
> > for a single origin...... No idea about plant vacuole origins....
> >
> > @dosumis <https://github.com/dosumis> so which type of endosome isn't a
> > transport vesicle (endocytosis is_a vesicle-mediated transport?)
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > 
> <https://github.com/geneontology/go-ontology/issues/12264#issuecomment-231057031>,
> > or mute the thread
> > 
> <https://github.com/notifications/unsubscribe/AAG4x-lFlGuAevUbu6m_CQFuy3FjYkzTks5qTOj8gaJpZM4HUNNW>
> > .
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/geneontology/go-ontology/issues/12264#issuecomment-231064498>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AHBLKH2M9yCXlFFUXUp0G6kgmia3-olmks5qTPIqgaJpZM4HUNNW>.
>


-- 
Cambridge University
PomBase http://www.pombase.org/
Cambridge Systems Biology Centre http://www.sysbiol.cam.ac.uk/Investigators/val-wood

**Comment by ValWood** [comment]
On 07/07/2016 13:42, David Osumi-Sutherland wrote:
> So maybe general vacuole should be parent to lysosome, not endosome. 
That's how it used to be...

> How
> does the term lytic vacuole fit?


-- 
Cambridge University
PomBase http://www.pombase.org/
Cambridge Systems Biology Centre http://www.sysbiol.cam.ac.uk/Investigators/val-wood

**Comment by deustp01** [comment]
Looking at Wikipedia (which appears to be the source of the GO vesicle and vacuole definitions), here's the start of the entry for vacuole:

Not to be confused with vesicle (biology and chemistry).
A vacuole (/ˈvækjuːoʊl/) is a membrane-bound organelle which is present in all plant and fungal cells and some protist, animal[1] and bacterial cells.[2] Vacuoles are essentially enclosed compartments which are filled with water containing inorganic and organic molecules including enzymes in solution, though in certain cases they may contain solids which have been engulfed. **Vacuoles are formed by the fusion of multiple membrane vesicles and are effectively just larger forms of these** (emphasis added).[3] The organelle has no basic shape or size; its structure varies according to the needs of the cell.

If that sentence about the formation of vacuoles by vesicle fusion is correct for yeasts and plants, then all of this supports the idea that "vesicle" should be a parent term with vacuole, lysosome, endosome, secretory granule, etc. as is_a children, on the basis of their common structural features. I'm ducking the issue of what additional relationships there should be among these children, e.g., can an endosome be transformed into a late endosome and from there to a lysosome, because my knowledge, especially for non-animal cells, is very limited.
My sense is that it's not Val who's being sloppy here but animal biologists who sometimes use the terms interchangeably (David H pointed to a group of Reactome text summaries that do this).
So the questions may be whether it's good biology to treat yeast vacuoles as a specialized kind of vesicle and, if so, whether the relevant biologists would accept this usage.

**Comment by deustp01** [comment]
Continuing the thought, I guess lytic vacuole is_a vacuole is_a vesicle. The definition of lytic vacuole is "A vacuole that is maintained at an acidic pH and which contains degradative enzymes, including a wide variety of acid hydrolases." so that makes lysosome one of its children (i.e., that part of the hierarchy looks right).

**Comment by ukemi** [comment]
So is the solution to move endosome out from under vacuole and back to membrane bound vesicle?

-D

**Comment by deustp01** [comment]
But, from discussion above, does vacuole have a specific meaning for yeasts (and plants?) that interferes with this? Are taxon constraints needed? Which is a long way of saying that if this sounds OK to Val, no problems here.

**Comment by dosumis** [comment]
Should be an membrane bound organelle at least.

**Comment by deustp01** [comment]
But a vesicle is_a membrane-bound organelle. The definition of GO:0031982 vesicle is "Any small, fluid-filled, spherical organelle enclosed by membrane." It has two children for more specific kinds of vesicles, "extracellular ~" and "intracellular ~", and some part children, as well as GO:0031988 membrane-bounded vesicle. What distinctive role does this last term have, except perhaps to annotate vesicles described in experiments done in such a way that its location with respect to other cellular structures is unknown?

**Comment by dosumis** [comment]
Guess I'm just going with my gut feeling for what vesicle means. Something like: a small ~spherical membrane bound organelle that transports stuff. 

Feels wrong to use it for all intracellular organelles with a single membrane. I bet we have plenty of those that are not under 'vesicle'.  Plus pretty sure endosome topography can be quite complex.

**Comment by paolaroncaglia** [comment]
Wrt GO:0031988 membrane-bounded vesicle, a clarification: all vesicles are bounded by membranes, this was confirmed in discussion with experts and editors, and the relevant edits are on my to-do list, specifically the merging of 'membrane-bounded vesicle' with 'vesicle’, see bottom of https://github.com/geneontology/go-ontology/issues/11504.

**Comment by ValWood** [comment]
I'm happy for any solution which stops my endosomes being annotated to vacuole ;)

**Comment by deustp01** [comment]
Sorry, Paola, I missed / forgot that discussion, so you're ahead of me.
Peter

**Comment by ukemi** [comment]
Coming back to this one. I think the safest thing is to move endosome to be a child of cytoplasmic, membrane-bounded vesicle. The endosome terms should sort themselves when the reasoner runs. Correct?

**Comment by paolaroncaglia** [comment]
Note about membrane-bounded vesicle - those are indeed just vesicles, I'll do that merge tomorrow. Then David can place endosome under cytoplasmic vesicle.

**Comment by ukemi** [comment]
Sounds good. Thanks Paola.

**Comment by paolaroncaglia** [comment]
@ukemi as you know, 'membrane-bounded vesicle' has finally been merged into its parent 'vesicle'. Same for cytoplasmic vesicles. Thank you for your help with fixing the broken build.

**Comment by ukemi** [comment]
Endosome is now a cytoplasmic vesicle.
